modules/core/exported/client.go

```diff
@@ -154,6 +169,8 @@ type ConsensusState interface {
 	GetTimestamp() uint64
 
 	ValidateBasic() error
+
+	GetRoot() Root
```

Comment:
Had to add back this interface method so that multi-hop doesn't need to directly depend on tendermint client.